### ISSUE TITLE
All sections can be submitted (epic)

### ISSUE THREAD

**Issue by cjhskippy** [18F/cms-carts-seds#416]
Collection of the open issues related to displaying the content within each section and accepting user input.  Does not include tickets specifically addressing prior year data.

**Basic Info (section 0):**
  - [ ] #613 - Pre-filled info for each state
  - verify status of DSCP edits and testing.  Complete?

**Section 1:**
  - [ ] DSCP edits pending (PR #619) 
  - [ ] Test section after edits

**Section 2:**
 - [ ] #477 (show ACS data)
 - [ ] #639 - Table headings (bug)
 - [ ] #639 - FY19 + FY20 SEDS data for all states
 - [ ] Change instructions for refreshing SEDS data
 - [ ] #641 - Show Part 1 Q1 and Part 2 Q2 that are currently conditionally-managed.  Also show skip text when applicable.
 - [ ] DSCP edits pending (PR #619)
 - [ ] Test section after edits

**Section 3:**
 - [ ] Fix navigation menu heading ("null")
 
**Section 3a:**
 - verify status of DSCP edits and testing.  Complete?

**Section 3b:**
 - verify status of DSCP edits and testing.  Complete?

**Section 3c:**
 - [ ] #315 - Section 3c build (in progress)
 - [ ] Provide support for synthesized values and tables. (PR #621)
 - Verify status of DSCP edits
 - [ ] Test section

**Section 3d:**
 - Verify status of DSCP edits
 - Verify testing status

**Section 3e:**
 - Verify status of DSCP edits
 - Verify testing status

**Section 3f:**
 - Verify status of DSCP edits
 - Verify testing status

**Section 3g:**
- [ ] #356 - JSON for section 3G
- [ ] #468 - Fixtures for section 3G
- [ ] #211 - Section 3g build
- DSCP edits
- Test

**Section 3h:**
- [ ] #212 - Section 3h build (in progress)
- [ ] #408 - Working upload component
- [x] #566 
- [x] #567 
- [ ] #578
- DSCP edits
- Test after completion of all above items

**Section 3i:**
- [ ] #382 - Section 3i build 
  - [ ] repeatable for HSI programs (add program, remove program)
  - [ ] percentage for part 2 question 5 is automatically calculated _for the correct HSI program in the repeatable_
  - [ ] upload associated _with the correct HSI program in the repeatable instance_ (blocked by #408)
- Ensure DSCP edits are incorporated
- Test after completion of all above items

**Section 4:**
 - [ ] #447 - Frontend build displays
 - [ ] #380 - Move from 2b to 4 (likely duplicate of #447)
 - [ ] #606 - Move JSON for section 2b to section 4
 - [ ] DSCP edits pending (PR #619).  Verify this contains all of the edits.
 - [ ] #408 - Working upload component
 - [ ] Test

**Section 5:**
 - [ ] #381 - Section 5 build
   - [ ] Support for section 5's synthesized table
 - [ ] Enable section display

**Section 6:**
 - DSCP edits and testing status?
 - [ ] #385 - Certify and submit build

Note: follow up on #383 re: section organization - either close or adopt reorganization.

Some sections that have already been built need to have a handful of components connected to the redux store:
- [ ] #418 - PageInfo component
- [ ] #436 - Percentage Calculator component
- [x] #539 - Definitions / Help Drawer

Also we need to save to the database, which is blocked pending user authentication.
- [ ] #475 - Form saves to database
